client/my-sites/themes/themes-selection.jsx

```diff
@@ -168,18 +163,18 @@ const ConnectedThemesSelection = connect(
 			themesCount: getThemesFoundForQuery( state, siteIdOrWpcom, query ),
 			isRequesting: isRequestingThemesForQuery( state, siteIdOrWpcom, query ),
 			isLastPage: isThemesLastPageForQuery( state, siteIdOrWpcom, query ),
-			isThemeActive: themeId => isThemeActive( state, suffixThemeId( themeId ), siteId ),
+			isThemeActive: themeId => isThemeActive( state, themeId, siteId ),
 			isThemePurchased: themeId => (
 				// Note: This component assumes that purchase and data is already present in the state tree
 				// (used by the isThemePurchased selector). At the time of implementation there's no caching
 				// in <QuerySitePurchases /> and a parent component is already rendering it. So to avoid
 				// redundant AJAX requests, we're not rendering the query component locally.
-				isThemePurchased( state, suffixThemeId( themeId ), siteId ) ||
+				isThemePurchased( state, themeId, siteId ) ||
```

Comment:
I'm pretty sure this is a bug in our initial implementation, since there is no jetpack API for querying purchases, meaning all purchase queries will use original (non-suffixed) ids.